plugins/modules/kubevirt_vm_info.py

```diff
@@ -194,6 +212,7 @@ def arg_spec():
         "namespace": {},
         "label_selectors": {"type": "list", "elements": "str", "default": []},
         "field_selectors": {"type": "list", "elements": "str", "default": []},
+        "running": {"type": "bool", "default": True},
         "wait": {"type": "bool", "default": False},
         "wait_sleep": {"type": "int", "default": 5},
         "wait_timeout": {"type": "int", "default": 120},
```

Comment:
Let's add the following to arg spec definition to validate the param data passed to ensure the wait condition is provided when `running` is provided.

```python
        required_by=dict(
            wait=('running', ),
        ),
```

Reply:
Addressed!